app/src/main/java/com/github/studydistractor/sdp/FireBaseProcrastinationActivityService.kt

```diff
@@ -0,0 +1,37 @@
+package com.github.studydistractor.sdp
+
+import android.util.Log
+import androidx.compose.runtime.mutableStateListOf
+import androidx.compose.runtime.snapshots.SnapshotStateList
+import com.google.firebase.database.DataSnapshot
+import com.google.firebase.database.DatabaseError
+import com.google.firebase.database.DatabaseReference
+import com.google.firebase.database.FirebaseDatabase
+import com.google.firebase.database.ValueEventListener
+import com.google.firebase.database.ktx.database
+import com.google.firebase.ktx.Firebase
+
+class FireBaseProcrastinationActivityService : ProcrastinationActivityService{

```

Comment:
This should probably be called FirebaseProcrastinationService, it should not be bound to an activity.

Reply:
As you said in willy's review